snapcraft.yaml

```diff
@@ -2,27 +2,41 @@ name: pi
 version: 20-1
 summary: Raspberry Pi gadget
 description: |
- Support files for booting Raspberry Pi.
- This gadget snap supports the Raspberry Pi 2B, 3B, 3A+, 3B+, 4B, Compute
- Module 3, and the Compute Module 3+ universally.
+  Support files for booting Raspberry Pi.
+  This gadget snap supports the Raspberry Pi 2B, 3B, 3A+, 3B+, 4B, Compute
+  Module 3, and the Compute Module 3+ universally.
 type: gadget
 base: core20
 architectures:
   - build-on: [amd64, armhf]
     run-on: armhf
+
+package-repositories:
+  - type: apt
+    ppa: snappy-dev/image
+
 confinement: strict
 grade: stable
 parts:
   gadget:
     plugin: nil
+    source: .
     override-build: |
+      set -x
+      if [ "$SNAPCRAFT_ARCH_TRIPLET" = "x86_64-linux-gnu" ] || [ -z "$SNAPCRAFT_ARCH_TRIPLET" ]; then
```

Comment:
Why is this if block needed? Surely both in native, and cross case, BUILD_ARCH_TRIPLET must have the same value, no?

Reply:
fair we probably can just unconditionally set BUILD_ARCH_TRIPLET like this, I will test this out

Reply:
this works to set it unconditionally